src/common/types/chainner-scope.ts

```diff
@@ -102,6 +102,12 @@ def BorderType::getOutputChannels(type: BorderType, channels: uint) {
         _ => channels
     }
 }
+def FpMode::toString(mode: FpMode) {
```

Comment:
Could you also use this function in the 2 other places? Just search for uses of `FpMode::fp32` in our code base and you'll find them.